net/sing-box/Makefile

```diff
@@ -78,7 +78,7 @@ define Package/sing-box/config
 
 		config SINGBOX_WITH_UTLS
 			bool "Build with uTLS support for TLS outbound"
-			default y
+			default n
```

Comment:
```suggestion
```

Just remove the `default y`; `default n` is redundant and should be avoided (openwrt/openwrt@8bc72ea7be3976711dacc09f0fdab061d6e5152a).  There has not been a treewide removal in the packages feed _yet_, but I think we should follow the example.

Reply:
I don't think that should change in this PR.

Reply:
@brvphoenix Do you think we should make this change in the PR?

Reply:
> I don't think that should change in this PR.

What do you mean by that?  That you should not change the `default y` that's currently there?
The config should definitely not have `default n` lines anywhere.  You may as well take the rest of them out, but please don't add another one.